frontend/ui/widget/inputtext.lua

```diff
@@ -671,13 +672,21 @@ end
 --- Search for a string.
 -- if start_pos not set, starts a search from the next to cursor position
 -- returns first found position or 0 if not found
-function InputText:searchString(str, start_pos)
+function InputText:searchString(str, case_sensitive, start_pos)
     local str_len = string.len(str)
     local char_pos, found = 0, 0
+    local char_txt
```

Comment:
Could be declared `local` just where it is used, inside the loop ?
(It just feels odd to have it declared that far away :) and you'd assume it has a larger scope.)

Reply:
Thanks, improved my knowledge of the scope of local variables.